### DISCUSSION TITLE
Linking to Bullhorn platform.

### DISCUSSION THREAD

**Discussion by benembery** [bullhorn/career-portal#449] [Q&A] [answered]
I've been directed here from technical support to ask a question about creating links into the Bullhorn UI. 

My client has a bespoke job submission integration within their wesbite. We send notification emails from the website whenever a job submission is submitted. It would be fantastic if I could link directly to the Bullhorn UI from the email. 

Here is the link I was attempting to use, that does not seem to work. 

```
https://redacted.bullhornstaffing.com/BullhornStaffing/OpenWindow.cfm?Entity=JobSubmission&ID=347
```

If I replace `ID` with the `Job ID` and `Entity` with `JobOrder` in the URL above the link works as expected, sadly is doesn't work with the JobSubmission ID. Is there any reason why this would be the case?

**Comment by charlesabarnes** [comment, marked as answer]
Heya,  this is probably was not the right place for you to be directed to, but submissions do not have an overview page to be linked to.  You want to either redirect to the Candidate or the Job pages

**Reply by benembery** [reply]
Thanks!
